### PR TITLE
Removed the header

### LINKED ISSUE #2669: [Portuguese] Update the style guide to reflect the removal of the "What is it" header
## Description
Update the style guide to reflect the removal of the "What is it" header
Ptbr page: https://glossary.cncf.io/pt-br/style-guide/
Original page: https://glossary.cncf.io/style-guide

## References
https://github.com/cncf/glossary/issues/2667
https://github.com/cncf/glossary/issues/2242

### PR DESCRIPTION
**Pull request by professorabhay**
[pt] Updated the style guide to reflect the removal of the "What is it" header

### resolves #2669 

- [x] This PR does not contain plagiarism
- [x] I have signed off on all commits

### CODE REVIEW

**Comment by professorabhay** [comment]
Hey, @edsoncelio ! 
Finally, here is my PR 😮‍💨
Please take a look and let me know if you are looking for further changes 🙂

**Comment by edsoncelio** [comment]
> Hey, @edsoncelio ! Finally, here is my PR 😮‍💨 Please take a look and let me know if you are looking for further changes 🙂

Thanks! 
As soon as possible (this week) I'll take a look
/cc @MrErlison

File: content/pt-br/style-guide/_index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ category: Conceito
 
 As definições para **tecnologia** e **conceito** contém três subcabeçalhos:
```
Comment (edsoncelio) [inline content/pt-br/style-guide/_index.md:101]: "As definições para tecnologia e conceito contém dois subcabeçalhos:"

File: content/pt-br/style-guide/_index.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ category: Conceito
...
 
-- **O que é**: forneça uma visão curta e clara da definição abordada.
+- forneça uma visão curta e clara da definição abordada.
```
Comment (edsoncelio) [inline content/pt-br/style-guide/_index.md:103]: This line can be removed

File: content/pt-br/style-guide/_index.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ Quando apropriado, use **exemplos do mundo real** que ajudam os leitores (especi
 
 Quando usado na sua definição, sempre aponte para termos existentes do glossário (apenas a primeira menção deve ser um *link*).
 
-**Exemplo**: veja seção "O que é" da definição de [service mesh](/service-mesh/). Ele se refere aos conceitos de microsserviços, serviços, confiabilidade e observabilidade. Além disso, é usado um exemplo do mundo real comparando desafios de rede em um ambiente de microsserviços (algo que pessoas não técnicas pode não ter conhecimento prévio) a problemas de *wifi* (algo que qualquer pessoa usando um computador pode entender). Sempre que possível tente fazer esse tipo de conexão. 
+**Exemplo**: veja seção da definição de [service mesh](/service-mesh/). Ele se refere aos conceitos de microsserviços, serviços, confiabilidade e observabilidade. Além disso, é usado um exemplo do mundo real comparando desafios de rede em um ambiente de microsserviços (algo que pessoas não técnicas pode não ter conhecimento prévio) a problemas de *wifi* (algo que qualquer pessoa usando um computador pode entender). Sempre que possível tente fazer esse tipo de conexão.
```
Comment (edsoncelio) [inline content/pt-br/style-guide/_index.md:118]: "veja seção da definição de [service mesh](/service-mesh/)" can be "veja a seção inicial da definição de [service mesh](/pt-br/service-mesh/)"

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi @professorabhay 
A few changes, after this will be LGTM

**Comment by professorabhay** [comment]
Ok @edsoncelio, I will commit the required changes asap. 
Btw sorry for the delay!

**Comment by edsoncelio** [comment]
Hey @professorabhay 
Do you need some help with the requested changes? 
If yes let me know :)

**Comment by edsoncelio** [comment]
Due to inactivity, I'll close this PR and continue working on another one. Thanks for your contribution.

**Pull request closed without merging**